public/app/plugins/datasource/cloudwatch/migration.test.ts

```diff
@@ -268,6 +269,7 @@ describe('migration', () => {
       const query = migrateVariableQuery(oldQuery);
       expect(query.region).toBe('us-east-1');
       expect(query.attributeName).toBe('rds:db');
+      expect(query.dimensionFilters).toStrictEqual({ InstanceId: '$instanceid' });
       expect(query.ec2Filters).toStrictEqual({ environment: ['$environment'] });
       expect(query.resourceType).toBe('elasticloadbalancing:loadbalancer');
       expect(query.tags).toStrictEqual({ 'elasticbeanstalk:environment-name': ['myApp-dev', 'myApp-prod'] });
```

Comment:
in theory, you should only test one thing at a time. Can you add a different test for your fix here?

Reply:
separated them out into different tests